### PR TITLE
destroy_vm.py tweaks

### PR DESCRIPTION
**Pull request by ddcrjlalumiere**
* Allow specifying VM by inventory name
* Really ignore certificate errors
* Provide information on arguments passed if no VM is found. Useful if wrapping in a loop.

### CODE REVIEW

File: samples/destroy_vm.py [hunk 1]
```diff
@@ -41,27 +43,50 @@ def setup_args():
...
+                        help='VM name of the VirtualMachine you want '
+                             'to destroy.')
 
     my_args = parser.parse_args()
 
     return cli.prompt_for_password(my_args)
 
+def get_obj(content, vimtype, name):
+    obj = None
+    container = content.viewManager.CreateContainerView(
```
Comment (tianhao64) [inline samples/destroy_vm.py]: I think we need to destroy the view afterwards.

File: samples/destroy_vm.py [hunk 2]
```diff
@@ -41,27 +43,50 @@ def setup_args():
...
                              'destroy')
+    parser.add_argument('-v', '--vm',
+                        help='VM name of the VirtualMachine you want '
+                             'to destroy.')
 
     my_args = parser.parse_args()
 
     return cli.prompt_for_password(my_args)
 
+def get_obj(content, vimtype, name):
```
Comment (tianhao64) [inline samples/destroy_vm.py]: can we reuse helper methods from filter_vms.py to achieve this?

File: samples/destroy_vm.py [hunk 3]
```diff
@@ -41,27 +43,50 @@ def setup_args():
...
+            break
+
+    return obj
 
 ARGS = setup_args()
 SI = None
 try:
+    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+
     SI = connect.SmartConnect(host=ARGS.host,
```
Comment (tianhao64) [inline samples/destroy_vm.py]: Recommend to use connect.SmartConnectNoSSL instead.

File: samples/destroy_vm.py [hunk 4]
```diff
@@ -72,7 +97,7 @@ def setup_args():
     VM = SI.content.searchIndex.FindByIp(None, ARGS.ip, True)
 
 if VM is None:
-    raise SystemExit("Unable to locate VirtualMachine.")
+    raise SystemExit("Unable to locate VirtualMachine. Arguments given: vm - %s , uuid - %s , name - %s , ip - %s" % (ARGS.vm, ARGS.uuid, ARGS.name, ARGS.ip))
```
Comment (tianhao64) [inline samples/destroy_vm.py]: Can we use the new string format style, like the one blow?

**Comment by ddcrjlalumiere** [comment]
@tianhao64 Sure, I will take a crack at the suggested changes. Thanks.

**Comment by ddcrjlalumiere** [comment]
@tianhao64 Integrated your suggestions, with the exception of using methods from filter_vms.py which was proving vexing for me.

**Pull request merged**